packages/fast-components-react-base/src/select/select.props.ts

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import {
+    ManagedClasses,
+    SelectClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { ListboxItemProps } from "../listbox-item";
+import { SelectState } from "./select";
+
+export interface SelectManagedClasses extends ManagedClasses<SelectClassNameContract> {}
+export interface SelectUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface SelectHandledProps extends SelectManagedClasses {
+    /**
+     * The children populate the select menu, any SelectOption components in the
+     */
+    children?: React.ReactNode | React.ReactNode[];
+
+    /**
+     * Custom function to render the base display of the control
+     */
+    contentDisplayRenderFunction?: (
+        props: SelectProps,
+        state: SelectState
+    ) => React.ReactNode;
+
+    /**
+     * Function which converts and formats the string may be displayed in the ui
+     * in the UI
+     */
+    displayStringFormatterFunction?: (
+        selectedOptions: ListboxItemProps[],
+        placeholder: string
+    ) => string;
+
+    /**
+     * Selected option id's (controlled mode)
+     */
+    selectedItems?: string[] | ListboxItemProps[];
+
+    /**
+     * Id's that are initially selected
+     */
+    defaultSelection?: string[] | ListboxItemProps[];
+
+    /**
+     * Specifies that the drop-down list is open
+     */
+    isMenuOpen?: boolean;
+
+    /**
+     * Specifies that the control is disabled
+     */
+    disabled?: boolean;
+
+    /**
+     * Defines one or more forms the select field belongs to
+     */
+    form?: string;
+
+    /**
+     * Specifies that multiple options can be selected at once
+     */
+    multiselectable?: boolean;
+
+    /**
+     * Defines a name for the drop-down list
+     */
+    name?: string;
+
+    /**
+     * String displayed when there is no selected value
+     */
+    placeholder?: string;
```

Comment:
I think placeholder maps to an unhandledProp on HTMLDivElement

Reply:
What do you suggest as an alternative?

Reply:
Discussed w. Jane, decided we should keep the prop names that match the html select element props.